app/models/movie.rb

```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
 
 class Movie < ApplicationRecord
+  has_many: screenings, dependent: :destroy
+  validates_associated :screenings
+  validates :duration_min , numerically: {greater_than: 1}
```

Comment:
I think just the duration would be enough. I think you can document this in some kind of ADR or documentation of what this is.

Reply:
yes